### ISSUE TITLE
Add SDTM examples

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/examples#36]
SDTM examples will get added later once sdtm.oak open sourced - and will combine with examples for datacutr and sdtmchecks

**Comment by bms63** [comment]
what about linking pharmaversesdtm as well? 
![image](https://github.com/pharmaverse/examples/assets/10111024/44ffca3f-b266-42b5-8dc3-3a6f2ae61d86)

**Comment by rossfarrugia** [comment]
@bms63 that one's already mentioned on the home page as used in most examples. for the SDTM examples in particular though they'll need to use raw synthetic data. there is a component the oak team have here that could help that step.

**Comment by rossfarrugia** [comment]
@rammprasad i sent you write access to this repo and you can self-assign yourself or delegate this once sdtm.oak is ready

**Comment by rossfarrugia** [comment]
Let's prioritise DM and AE here as requested for the examples Phil from Posit has been using for workshops.

**Comment by bms63** [comment]
Are we talking about creating DM and AE datasets from raw data?

**Comment by rossfarrugia** [comment]
Yes using sdtm.oak and raw synthetic data the oak team have created. Let's chat on Friday about Phil's request as hopefully we can enable a couple of e2e examples that enable running SDTM -> ADaM -> TLG using the same consistent synthetic data, so that any workshop attendees get continuity here when trying out packages for each clinical reporting step.

**Comment by bms63** [comment]
added!!

**Comment by rossfarrugia** [comment]
Thanks @ShiyuC - as part of this PR please can you add `{pharmaverseraw}` to this sentence on the site homepage: _The examples use consistent source test SDTMs and ADaMs from {pharmaversesdtm} and {pharmaverseadam} respectively._
